docs/Common/Structs/index.md

```diff
@@ -1130,3 +1130,9 @@ There are no defined parameters for this struct
 |hybridAppPreference|[Common.HybridAppPreference](../enums/#hybridapppreference)|false||Specifies the user preference to use one specific app type or all available types|
 |endpoint|String|false|maxlength: 65535|If specified, which Core uses a client implementation of the connection type and attempts to connect to the endpoint when this app is selected (activated).<br>If omitted, Core won't attempt to connect as the app selection (activation) is managed outside of Core. Instead it uses a server implementation of the connection type and expects the app to connect|
 
+### GearStatus
```

Comment:
```suggestion
### GearStatus

```

Reply:
fixed in 78995a7a3f364d0a93ec57cb1b5b59c0099dffb6